### PR TITLE
Support deploying CF templates to stacks

### PR DESCRIPTION
**Pull request by djmitche**
This supports templates written cfn_pyplates, which is a Python DSL.  It's pretty straightforward, and anyway well-documented.  But it has the advantage of supporting comments (which JSON does not) and being executable (so, for example, we can use loops and whatnot where necessary).

Templates are tied to stacks and parameters with `stacks.yml`.

The script supports creating, updating, and deleting stacks, and can show the "events" from CloudFormation during each operation, with the `--wait` option.

The template itself is just a placeholder, so don't worry about the bogus CidrBlock.

### CODE REVIEW

**Comment by djmitche** [comment]
Travis should be OK now.  @rail any comments?  I'll work on adding some tests.

File: scripts/aws_deploy_stack.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+def main():
+    stacks_yml = os.path.abspath(
+        os.path.join(
+            os.path.dirname(__file__),
+            "../configs/cloudformation/stacks.yml"))
+    parser = argparse.ArgumentParser(description='deploy a cloudformation stack')
+    parser.add_argument('stack', type=str,
+                        help='CloudFormation stack to create or update')
+    parser.add_argument('--config', type=str,
```
Comment (rail) [inline scripts/aws_deploy_stack.py]: type=argparse.FileType('r') maybe? So it fails early if file doesn't exist.

**Comment by rail** [comment]
It looks great to me!

File: scripts/aws_deploy_stack.py (hunk 1 above)
Comment (djmitche) [inline scripts/aws_deploy_stack.py]: I think it's hard to add a default in that case..

**Comment by djmitche** [comment]
I added some tests, but just of the `main` function.  As long as the coverage keeps going up :)

**Pull request merged**